gateway-service-definitions/src/main/java/org/apache/knox/gateway/service/definition/ServiceDefinition.java

```diff
@@ -21,6 +21,10 @@
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlElementWrapper;
 import javax.xml.bind.annotation.XmlRootElement;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
```

Comment:
Do we need Commons lang3 here for to string?

Reply:
Why not? I like it...

Reply:
Kinda the same reason as guava. If its not a critical dependency we don't really need it.